### PR TITLE
Fixed code reloader

### PR DESCRIPTION
**Pull request by DannySantos**
Reloader and server now run on different processes. Start
reloader with `bundle exec hanami reloader`

### CODE REVIEW

File: .hanami.server.guardfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+url = ENV.fetch('CONTROL_URL')
```
Comment (Swiftda01) [inline .hanami.server.guardfile]: I can't see where this is being used

File: .hanami.server.guardfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+token = ENV.fetch('CONTROL_TOKEN')
```
Comment (Swiftda01) [inline .hanami.server.guardfile:1]: Will I need to set this locally in order for the reloader to work?

File: README.md [hunk 3]
```diff
@@ -22,6 +22,12 @@ How to run the development server:
 % bundle exec hanami server
 ```
 
+How to run the code reloader:
+
+```
+% bundle exec hanami reloader
```
Comment (Swiftda01) [inline README.md:28]: Perhaps we should make it clear that the server needs to be running in order for this command to work

**Review by Swiftda01 — COMMENTED** [review]
If the expected behaviour here is for changes to the app to automatically take affect without having to restart the server (or re-run the reloader command) then this doesn't appear to be working for me, unless I have misunderstood the purpose here?

File: README.md (hunk 3 above)
Comment (DannySantos) [reply README.md:28]: It doesn't, sounds like there's only something wrong in your environment. Are you getting any errors?

**Pull request merged**